native/CMakeLists.txt

```diff
@@ -376,10 +376,6 @@ elseif(MACOSX)
     cocos_source_files(
         cocos/main/main-mac.mm
     )
-elseif(IOS)
-    cocos_source_files(
-        cocos/main/main-ios.mm
```

Comment:
This file should be remove too?

Reply:
Yes, I plan to put all the main in the template later. This allows users to extend the functionality of the platform。
[issue](https://github.com/cocos/cocos-engine/issues/10510)